src/blockchain/process-block.ts

```diff
@@ -1,39 +1,37 @@
 import Augur from "augur.js";
 import * as Knex from "knex";
 import { logError } from "../utils/log-error";
-import { Int256, BlocksRow } from "../types";
+import { Block, BlocksRow, Int256 } from "../types";
```

Comment:
This actually works/compiles and you get type info for `Block`?  I don't see you exporting `Block` from `types.ts` below so this working surprises me...